### PR TITLE
Correct u2f-api.js url in u2f registration page

### PR DESCRIPTION
**Pull request by MohammedEssehemy**
handle wrong script url which cause 404 error in user_profile page

### CODE REVIEW

**Comment by MohammedEssehemy** [comment]
any updates here?

File: modules/jaggery-apps/portal/gadgets/user_profile/js/gadget.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 function drawPage() {
     var output = "";
-    var u2fScript="<script src=\"u2f-api.js\"></script>";
+    var u2fScript="<script src=\"/portal/gadgets/user_profile/js/u2f-api.js\"></script>";
```
Comment (ruwanta) [inline modules/jaggery-apps/portal/gadgets/user_profile/js/gadget.js]: The problem in adding url starting with "/" prevent the portal to be re-mapped to different context path via reverse-proxy.
The idea of having the relative URL based on the current page (i.e /portal/gadgets/user_profile/js/")  makes it available even the context path is remapped.
@MohammedEssehemy , can you explain what was to motive to think the original URL is incorrect.

Comment (MohammedEssehemy) [reply modules/jaggery-apps/portal/gadgets/user_profile/js/gadget.js]: To handle wrong script url which cause 404 error in browser console in user_profile page

**Comment by IsurangaPerera** [comment]
From IS590 we'll be using WebAuthn API at the client-side of FIDO implementation. Furthermore, most browsers have dropped the support to U2F extension. 
Hence closing the issue.

@MohammedEssehemy appreciate your contributions to the project 👌

**Comment by GDRDABARERA** [comment]
Closing as this is related to the old APIs

**Pull request closed without merging**